android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
```

Comment:
areeb but dont you think now models should be consistent irrespective of the migrations we have to add? i mean if changes are need in migrations shouldnt be a problem aas long as attributes are consistent.